src/Plugin/Block/AlertBannerBlock.php

```diff
@@ -99,13 +140,29 @@ public function build() {
    *   Array with the IDs of any published alert banners.
    */
   protected function getCurrentAlertBanners() {
-    $published_alert_banner = $this->entityTypeManager->getStorage('localgov_alert_banner')
+    $types = $this->mapTypesConfigToQuery();
+    $published_alert_banner_query = $this->entityTypeManager->getStorage('localgov_alert_banner')
       ->getQuery()
       ->condition('status', 1)
       ->sort('type_of_alert', 'DESC')
-      ->sort('changed', 'DESC')
-      ->execute();
-    return $published_alert_banner;
+      ->sort('changed', 'DESC');
+    if (!empty($types)) {
+      $published_alert_banner_query->condition('type', $types, 'IN');
+    }
+    return $published_alert_banner_query->execute();
+  }
+
+  /**
+   * Get an array of types from config we can use for querying.
+   *
+   * @return array
+   *   An array of alter banner type IDs as keys and values.
+   */
+  protected function mapTypesConfigToQuery() : array {
+    $include_types = $this->configuration['include_types'];
+    return array_filter($include_types, function ($t) {
```

Comment:
Another way to write this would be ```array_filter($include_types)```

Reply:
If it works I'm ok leaving it for now. 
If it causes any problems we can open a new issue.